x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,529 @@
+[[update-node-certs]]
+== Updating node security certificates
+If your cluster certificates expired or you have a new Certificate Authority
+(CA), then you'll need to update the security certificates for both the
+transport and HTTP layers. When updating your node security certificates, there
+are two main scenarios that you'll encounter:
+
+* <<update-node-certs-same,Your certificate authority (CA) or key remains the same>>
```

Comment:
Usually the key _and_ certificate changes. It can happen that only the certificate changes and the key remains the same but this is not a common use case enough to name the category after it I think

Reply:
++ We can change these to read: 

> * <<update-node-certs-same,Your CA and key remains the same>>
> * <<update-node-certs-different,Your CA certificate and key changes>>